.changeset/clever-dolls-know.md

```diff
@@ -0,0 +1,7 @@
+---
+'@leafygreen-ui/side-nav': major
+'@leafygreen-ui/lib': minor
+'@leafygreen-ui/menu': patch
```

Comment:
As noted below, let's omit this altogether to keep from any conflicts with Brooke's PR.